sklearn/pipeline.py

```diff
@@ -771,7 +773,10 @@ class FeatureUnion(TransformerMixin, _BaseComposition):
         ``-1`` means using all processors. See :term:`Glossary <n_jobs>`
         for more details.
 
-    transformer_weights : dict, default=None
+        .. versionchanged:: v0.20
+           `n_jobs` changed from ``1`` to ``None``
+
+    transformer_weights : dict, optional
```

Comment:
please revert to default=None as it was before

Reply:
@NicolasHug I'm sorry but I don't understand what to do. In my code editor this piece looks like this now (I have merged changes from master and accepted your previous suggestion abot n_jobs):

```
    n_jobs : int, default=None
        Number of jobs to run in parallel.
        ``None`` means 1 unless in a :obj:`joblib.parallel_backend` context.
        ``-1`` means using all processors. See :term:`Glossary <n_jobs>`
        for more details.

        .. versionchanged:: v0.20
           `n_jobs` default changed from 1 to None

    transformer_weights : dict, optional
        Multiplicative weights for features per transformer.
        Keys are transformer names, values the weights.
```
What should I change here?

Reply:
It's usually easier to look at PR comments in the diff tab https://github.com/scikit-learn/scikit-learn/pull/16199/files

You'll see that you changed `transformer_weights : dict, default=None` to `transformer_weights : dict, optional` (probably during a merge?)

I suggest to use the previous version with default=None because that's what we use now.

Also it seems that some of the comments haven't been addressed yet ;)